.github/workflows/build.yml

```diff
@@ -40,8 +41,7 @@ jobs:
 
       - name: Set Artifact Name
         id: set-artifact-name
-        run: echo "artifact_name=$(echo "windows-${{ matrix.configuration }}" | tr '[:upper:]' '[:lower:]')" >> $GITHUB_OUTPUT
-        shell: bash
+        run: echo "artifact_name=$(echo "windows-${{ matrix.configuration }}" | tr '[:upper:]' '[:lower:]')" >> $GITHUB_ENV
```

Comment:
This breaks the build for Windows.

$GITHUB_OUTPUT is a specific var name 
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

Please restore this, other than that LGTM. Ready to merge once all pipelines pass and I get a chance to test on all platforms.